### PR TITLE
Asen Kraychev - Formatting code

### PR DESCRIPTION
**Pull request by akraychev**

### CODE REVIEW

File: formatting-code/a.kraychev/Manager.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<?php
+namespace League\CLImate\Settings;
+
+class Manager {
```
Comment (victorlazov) [inline formatting-code/a.kraychev/Manager.php:4]: Class indent in a namespace

File: formatting-code/a.kraychev/Manager.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return [];
+  }
+
+  public function importSetting($setting) {
+    $short_name = basename(
+      str_replace('\\', '/', get_class($setting))
+    );
+
+    $method = 'importSetting'
+      . $short_name;
```
Comment (victorlazov) [inline formatting-code/a.kraychev/Manager.php:53]: Unnecessary new line

File: formatting-code/a.kraychev/Manager.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      . $this->getClassName($name);
+  }
+
+  protected function getClassName($name) {
+    return ucwords(
+      str_replace('add_', '', $name)
+    );
+  }
+
+  public function settings() {
```
Comment (victorlazov) [inline formatting-code/a.kraychev/Manager.php:43]: Public methods should be on top of private/protected.

File: formatting-code/a.kraychev/JavaScript-code.js [hunk 4]
```diff
@@ -0,0 +1,126 @@
+b = navigator.appName;
+addScroll = false;
+
+if (navigator.userAgent.indexOf('MSIE 5') > 0
```
Comment (asparuh-krastev) [inline formatting-code/a.kraychev/JavaScript-code.js:4]: This should be in the same paragraph with addScroll = false; in my opinion

File: formatting-code/a.kraychev/JavaScript-code.js [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    || navigator.userAgent.indexOf('MSIE 6') > 0
+) {
+  addScroll = true;
+}
+
+var off = 0;
+var txt = "";
+var pX = 0;
+var pY = 0;
+document.onmousemove = mouseMove;
```
Comment (asparuh-krastev) [inline formatting-code/a.kraychev/JavaScript-code.js:14]: This should be either in separate paragraph or in one paragraph with the next if.

File: formatting-code/a.kraychev/JavaScript-code.js [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+  if (b == "Netscape") {
+    pX = evn.pageX - 5;
+    pY = evn.pageY;
+  }
+  else {
+    pX = event.x - 5;
+    pY = event.y;
+  }
+
+  if (b == "Netscape") {
```
Comment (asparuh-krastev) [inline formatting-code/a.kraychev/JavaScript-code.js:30]: I refactored this in one if-else

**Pull request merged**